.travis.yml

```diff
@@ -1,24 +0,0 @@
-language: go
-go: 1.12.x
-
-env:
-  global:
-    - GO111MODULE=on
-
-jobs:
-  include:
-    - os: linux
-      script: make
-    - os: osx
-      script: make
-      env:
-        - NO_DOCKER=true
-    - os: windows
-      script: go test -mod=vendor -parallel=1 -v ./...
-      after_success: go build -mod=vendor -o pack ./cmd/pack
-      env:
-        - NO_DOCKER=true
-
-branches:
-  only:
-    - master
```

Comment:
Is there any value in using a bimodal testing approach while we iterate towards Github Actions being the only way?

Reply:
Can't think of a single value keeping Travis around would provide. You?